resources/ts/Components/Integrations/Detail/CredentialsUitidv1Consumers.tsx

```diff
@@ -47,37 +49,59 @@ export const CredentialsUitidv1Consumers = ({
       <Heading className="font-semibold lg:min-w-60" level={4}>
         {t("details.credentials.uitid_v1")}
       </Heading>
-      <div className="flex flex-col gap-6 min-w-[40rem]">
-        <div>
-          <Heading className="font-semibold flex" level={4}>
-            {t("details.credentials.test")}
-          </Heading>
-          <CopyText>{uiTiDv1TestConsumer?.apiKey}</CopyText>
-        </div>
-        <div>
-          {uiTiDv1ProdConsumer && (
+      <div className="flex flex-col gap-4 min-w-[40rem]">
+        {type !== IntegrationType.Widgets && (
+          <>
+            <Heading className="font-semibold" level={4}>
+              {t("details.credentials.test")}
+            </Heading>
+            <CopyText>{uiTiDv1TestConsumer?.apiKey}</CopyText>
+          </>
+        )}
+
+        {status !== IntegrationStatus.Active &&
+          type === IntegrationType.Widgets && (
             <>
               <Heading className="font-semibold" level={4}>
-                {t("details.credentials.live")}
+                {t("details.credentials.test")}
               </Heading>
-              {status === "active" && (
-                <CopyText>{uiTiDv1ProdConsumer?.apiKey}</CopyText>
-              )}
-            </>
-          )}
-          <div className="flex flex-col gap-3 align-center">
-            {status !== "active" && <StatusLight status={status} />}
-            {status === "draft" && auth0Clients.length === 0 && (
-              <ActivationFlow
-                status={status}
-                id={id}
-                subscription={subscription}
+              <OpenWidgetBuilderButton
                 type={type}
-                email={email}
+                id={uiTiDv1TestConsumer!.apiKey}
               />
-            )}
-            <Alert variant="info">{t("details.credentials.info")}</Alert>
-          </div>
+            </>
+          )}
+        <div className="flex flex-col gap-2">
+          <Heading className="font-semibold" level={4}>
+            {t("details.credentials.live")}
+          </Heading>
+          <StatusLight status={status} />
+          {uiTiDv1ProdConsumer && (
```

Comment:
Wondering if this condition should be `status === Active` since both conditions within rely on it and the to should be equivalent to me. Then you can simplify the condition within the div to a ternary on `type === Widgets`